release-notes/latest.md

```diff
@@ -1,3 +1,12 @@
 #### <sub><sup><a name="4938" href="#4938">:link:</a></sup></sub> feature
 
 * Include job label in build duration metrics exported to Prometheus. #4976
+
+#### feature
+
+* Support path templating for secret lookups in Vault credential
```

Comment:
Feel free to credit yourself! :slightly_smiling_face: e.g. '@agurney added support for ...'

Most release notes do this but it's not necessary if you don't want to. We usually link to the PR at the end though so people can see more details (see above release note).

Reply:
I'll pop that in, a bit chicken/egg initially for knowing the PR number ;-)

Reply:
In it has popped.